pom.xml

```diff
@@ -11,14 +11,14 @@
   <packaging>jar</packaging>
   <groupId>org.webjars</groupId>
   <artifactId>bpmn-js</artifactId>
-  <version>0.10.4-SNAPSHOT</version>
+  <version>0.11.0-0-SNAPSHOT</version>
```

Comment:
Usually we start with just `0.11.0` and then if we need a bug fix we add `-1` so that the first version is consistent with the upstream.